draft-ietf-quic-qpack.md

```diff
@@ -444,33 +452,40 @@ referenced by a given relative index will change while interpreting instructions
 on the encoder stream.
 
 ~~~~~ drawing
-    +---+---------------+-----------+
-    | n |      ...      |   d + 1   |  Absolute Index
-    + - +---------------+ - - - - - +
-    | 0 |      ...      | n - d - 1 |  Relative Index
-    +---+---------------+-----------+
-      ^                       |
-      |                       V
-Insertion Point         Dropping Point
+      +---+---------------+-----------+
+      | n |      ...      |     d     |  Absolute Index
```

Comment:
If Absolute Index is zero-based, then the Absolute Index of the most recently inserted item would be n-1.